app/services/reinstate_declined_offer.rb

```diff
@@ -0,0 +1,16 @@
+class ReinstateDeclinedOffer
+  def initialize(course_choice:, zendesk_ticket:)
+    @course_choice = course_choice
+    @zendesk_ticket = zendesk_ticket
+  end
+
+  def save!
+    @course_choice.update!(
+      status: 'offer',
+      declined_at: nil,
+      declined_by_default: false,
+      decline_by_default_at: 10.days.from_now,
```

Comment:
`10.business_days.from_now`

Reply:
Just a thought, but do we need to reset the dbd dates for all the application choices if we're doing one @duncanjbrown ?

Reply:
I've just changed this spec as per Duncans [suggestion](https://github.com/DFE-Digital/apply-for-teacher-training/pull/4572#discussion_r618534268)

Reply:
So as per this discussion here https://ukgovernmentdfe.slack.com/archives/CP18YJXPY/p1619187153335900?thread_ts=1618916891.255500&cid=CP18YJXPY, we need to reset the dbd of all the applications on the application form for choices in the offer state.